modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/getLocalizedValue.ts

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import {NESTING_DELIMITER, NESTING_ROOT} from '../Constants';
```

Comment:
In our files structure proposal we didn't talk much about constants :thinking: Capital `C` is a tad weird, I'd expect a component behind it. Maybe it makes sense to bundle them with utils, `/utils/constnts.ts`?

Reply:
I was in doubt because DSM has a `Constants.tsx` but FDS has some `constants.ts` files. Makes sense, moving it to `/utils/constants.ts`